aries-site/src/examples/components/tooltip/ToolTipExitExample.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Box, Button, Tip, Text } from 'grommet';
+import { FormClose } from 'grommet-icons';
+
+export const ToolTipExitExample = () => {
+  return (
+    <Box align="center" justify="center" fill>
+      <Box gap="xsmall" round="xsmall">
+        <Box justify="between" direction="row" align="center">
+          <Text bold level={4} size="large">
+            Add Service
+          </Text>
+          <Tip dropProps={{ align: { left: 'right' } }} content="Close">
+            <Button icon={<FormClose />} />
```

Comment:
I picked the <FormClose /> because it aligned better but your right I wasn't sure which one.

Reply:
@britt6612 Can you follow up with @vavalos5 and find out the direction. The answer can be "in these situations, use <FormClose /> and in these other situations, use <Close />." Or the answer could be "we should look into it, I'll create an issue in the backlog"

Reply:
Sorry I didn't see your tags sooner @halocline. Whether to use `<Close />` or `<FormClose />` is a great question! Since this is not a form, we would be using `<Close />`. Here's the [modal example](https://www.figma.com/file/Ts53TAipMolmsv9DxWG3p0/HPE-Layer-Component?node-id=100%3A70) in layer for reference.

Reply:
Ill have to go back and change the layer modal example to use the `<Close />` icon. This icon does seem a lot larger then what the designs call for so when I set the size to `small` the alignment is a bit off

Reply:
The first image is with  `<Close /> size small
<img width="317" alt="Screen Shot 2021-02-24 at 2 42 51 PM" src="https://user-images.githubusercontent.com/42451602/109069531-93bdab00-76ae-11eb-9b8c-f54135a793a5.png">

The second image is with  `<Close /> default size 
<img width="344" alt="Screen Shot 2021-02-24 at 2 43 17 PM" src="https://user-images.githubusercontent.com/42451602/109069576-a33cf400-76ae-11eb-8e8b-70177e70627d.png">

Reply:
I would go with `small`

Reply:
Brittany and I discussed how the vertical alignment for `< Close />` looks terrible. Considering we were going to make this size` small`, we decided to use` <FormClose />` as it aligns much better.